src/components/Calculator.js

```diff
@@ -0,0 +1,138 @@
+import './Calculator.css';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const ShowResult = (props) => {
+  const { result } = props;
+  return (
+    <div className="result-container">
+      <p className="result">{result}</p>
+    </div>
+  );
+};
+
+ShowResult.defaultProps = { result: 0 };
+ShowResult.propTypes = { result: PropTypes.number };
+
+export const KeyCalc = (props) => {
+  const { eachKey, myClass } = props;
+  return (
+    <input type="button" className={`eachKey my-${eachKey} ${myClass}`} value={eachKey} />
+  );
+};
```

Comment:
- [ ] (Optional) You do not need to export this component since the `Calculator` component is rendering it and they are both in the same file. You would want to do a named export when the class/function/variable is in a file and you need it in another file. 👍